docs/sql-reference/sql-statements/data-definition/ALTER TABLE.md

```diff
@@ -325,7 +328,7 @@ You can add a generated column and specify its expression. [The generated column
 
 #### Modify table properties
 
-Currently, StarRocks supports modifying bloomfilter columns, colocate_with property, dynamic_partition property, enable_persistent_index property, replication_num property and default.replication_num property.
+Currently, StarRocks supports modifying bloomfilter columns, colocate_with property, dynamic_partition property, enable_persistent_index property, replication_num property and default.replication_num property, storage_cooldown_ttl property and storage_cooldown_time property.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/4832faf6-1983-40c7-9216-42581365d2cf)
Wrap all these properties in code format.